Grbl_Esp32/src/WebUI/WebServer.cpp

```diff
@@ -1223,11 +1223,7 @@ namespace WebUI {
             COMMANDS::wait(0);  //wdtFeed
         }
         file.close();
-        if (result) {
-            return SD.rmdir(path);
-        } else {
-            return false;
-        }
+        return result ? SD.rmdir(path) : false;
```

Comment:
Hmm, I'm not a fan of this construct, because `SD.rmdir` can have side effects. I'm not 100% sure if the standard guarantees that the lhs of this ternary operator is not executed if result evaluates to false, since function calls are not guaranteed to be ordered in expressions. (This actually got me in trouble more than I would care to admit)

If in doubt, best to just keep it safe and keep the 'if'.